tcms/telemetry/static/telemetry/js/testing/utils.js

```diff
@@ -1,10 +1,19 @@
 function loadInitialProduct(callback = () => {}) {
     jsonRPC('Product.filter', {}, data => {
-        updateSelect(data, '#id_product', 'id', 'name');
+        updateSelect(data, '#id_product', 'id', 'name', null);
         callback();
     });
 }
 
 function showOnlyRoundNumbers(number) {
     return number % 1 === 0 ? number : '';
 }
+
+// Close multiselect list when selecting an item
+// Iterate over all dropdown lists
+$("select:not([size])").each(function() {
+	$(this).on("change", function() {
+		// Clicking on the document closes the widget
+		$(document).click(); 
```

Comment:
what about `$(this).close()` ? Doesn't that work ?

Reply:
`$(this).close()` does not seem to work

Reply:
This is a bootstrap widget (e.g. there are more elements than meets the eye) and the open/close mechanics a re a bit different -> https://stackoverflow.com/questions/22346177/manually-close-bootstrap-dropdown-with-multiple-select

Based on that `$(this).parent('.bootstrap-select').removeClass('open')` works for me.

Reply:
Fixed in https://github.com/kiwitcms/Kiwi/pull/2441/files